### DISCUSSION TITLE
blog/your-first-stimulus-controller-learn-stimulus-ruby-on-rails-by-building-a-toggle-beginners-guide

### DISCUSSION THREAD

**Discussion by giscus** [harrison-broadbent/railsnotes-comments#7] [RailsNotes Comments]
# blog/your-first-stimulus-controller-learn-stimulus-ruby-on-rails-by-building-a-toggle-beginners-guide

So, you're new to Stimulus and want to try it out in your Ruby on Rails app. Or perhaps you just need a simple toggle pattern to follow. Either way, you're in the right place...

https://railsnotes.xyz/blog/your-first-stimulus-controller-learn-stimulus-ruby-on-rails-by-building-a-toggle-beginners-guide

**Comment by rverstappen** [comment]
Great, simple intro article. Thanks!

**Comment by salolivares** [comment]
Wow, great post! As a full time react dev, the stimulus way of doing things is neat!
